projects/storefrontlib/cms-components/navigation/search-box/search-box.component.ts

```diff
@@ -379,6 +465,71 @@ export class SearchBoxComponent implements OnInit, OnDestroy {
     }
   }
 
+  // Focus on previous item in results list
+  focusPreviousGroup(event: UIEvent) {
+    event.preventDefault(); // Negate normal keyscroll
+    const [results, focusedGroupIndex] = [
+      this.getGroupElements(),
+      this.getFocusedGroupIndex(),
+    ];
+
+    // Focus on last index moving to first
+    if (results[focusedGroupIndex].length) {
+      if (focusedGroupIndex < 1) {
+        results[0][0].focus();
```

Comment:
Not safe .focus() calling. results[0][0] could be undefined

Reply:
👍